src/bindings/jna/README.md

```diff
@@ -1,43 +1,121 @@
-A full java binding using JNA.
+A full java binding using JNA. This binding requires elektra to be installed on 
+the system to work.
 
 ## Usage ##
 
+To use the bindings in a java project, we have to include the jar file 
+libelektra-0.8.19.jar in the project. This jar is created upon build, if you 
```

Comment:
Not really changing much for the implementation, but maybe the style of `elektrify-*` is better suited. Then one can use `kdb run-java javaapp` (or similar for setting the CLASSPATH and so on).